x-pack/legacy/plugins/ml/public/application/jobs/jobs_list/components/delete_job_modal/delete_job_modal.js

```diff
@@ -17,160 +17,160 @@ import {
 
 import { deleteJobs } from '../utils';
 import { DELETING_JOBS_REFRESH_INTERVAL_MS } from '../../../../../../common/constants/jobs_list';
-import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
-
-export const DeleteJobModal = injectI18n(
-  class extends Component {
-    static displayName = 'DeleteJobModal';
-    static propTypes = {
-      setShowFunction: PropTypes.func.isRequired,
-      unsetShowFunction: PropTypes.func.isRequired,
-      refreshJobs: PropTypes.func.isRequired,
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+export class DeleteJobModal extends Component {
```

Comment:
I see this console warning when deleting a job. Is it related to your changes?

![image](https://user-images.githubusercontent.com/7405507/74045235-4e0b4300-49c4-11ea-94f3-f72d924d5567.png)

Reply:
I see this on master also. So not a new warning